doc/source/qip-simulator.rst

```diff
@@ -40,7 +40,7 @@ method.
 
 .. testcode::
 
-  from qutip import tensor
+  from qutip import tensor, basis
```

Comment:
This is likely because doctest does not create a new environment for every block or every rst file but run all the `testcode` from the first file to the last one. `basis` is imported before in other blocks (in another file). But we can leave it here, it makes it easier for someone just read this page. Thanks